lib/Remote/RemoteWebElement.php

```diff
@@ -69,10 +70,17 @@ public function clear()
      */
     public function click()
     {
-        $this->executor->execute(
-            DriverCommand::CLICK_ELEMENT,
-            [':id' => $this->id]
-        );
+        try {
+            $this->executor->execute(
+                DriverCommand::CLICK_ELEMENT,
+                [':id' => $this->id]
+            );
+        } catch (ElementNotInteractableException $e) {
+            // A long-standing bug in geckodriver/marionette prevents clicking on a link if the first child is a
+            // block-level element.
```

Comment:
What about adding reference to the bug in bugzilla? So that we can find the bug in the future to see if it is still present.